content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md

```diff
@@ -0,0 +1,42 @@
+---
+weight: 32
+title: Release 10.15.0.207
+layout: bundle
+---
+
+<!--10.15.199.0 - 10.15.0.207-->
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>The "Help and service" widget icons are no longer missing after a hard refresh.</td>

```

Comment:
are these two distinct widget icons? also, if it's UI texts, then it should probably be bold face, so `**Help** and **Service** widget icons`

Reply:
This refers to various icons in the "Help and service" widget. Perhaps better: In the "Help and service" widget, there are no longer missing icons after a hard refresh. ?